src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    static final SimpleDateFormat FORMATTER = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period ");
+        builder.append(dateFrom).append(" - ").append(dateTo);
+        try {
+            Date localDateFrom = FORMATTER.parse(dateFrom);
```

Comment:
[Use LocalDate instead of Date.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#use-localdate-instead-of-date)